src/components/PersonLink.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams, Link } from 'react-router-dom';
+import { Person } from '../types';
+
+const FATHER = 'father';
+const MOTHER = 'mother';
+const FEMALE = 'f';
+const NOT_SET_VALUE = '-';
+
+interface ParentLink {
+  person: Person,
+  parentType: string,
+  name: string,
+  peopleList: Person[],
+}
+
+const getParentLink = (
+  args: ParentLink,
+) => {
+  const {
+    person,
+    parentType,
+    name,
+    peopleList,
+  } = args;
+  const isMother = parentType === MOTHER;
+
+  const parent = peopleList
+    .find(personToFind => personToFind.name === name);
+
+  if (parent) {
+    return (
+      <Link
+        className={classNames(
+          { 'has-text-danger': isMother },
+        )}
+        to={`/people/${parent.slug}`}
```

Comment:
Do you need to add /people/? There is better solution to have slug there. It's was in theory videos